src/H5.c

```diff
@@ -91,30 +93,31 @@ static H5_atclose_node_t *H5_atclose_head = NULL;
 /* Declare a free list to manage the H5_atclose_node_t struct */
 H5FL_DEFINE_STATIC(H5_atclose_node_t);
 
-/*-------------------------------------------------------------------------
- * Function:    H5_default_vfd_init
- *
- * Purpose:     Initialize the default VFD.
- *
- * Return:      Success:        non-negative
- *              Failure:        negative
- *-------------------------------------------------------------------------
- */
-static herr_t
-H5_default_vfd_init(void)
+/*--------------------------------------------------------------------------
+NAME
+    H5__init_package -- Initialize interface-specific information
+USAGE
+    herr_t H5__init_package()
+RETURNS
+    Non-negative on success/Negative on failure
+DESCRIPTION
+    Initializes any interface-specific data or routines.
+--------------------------------------------------------------------------*/
+herr_t
+H5__init_package(void)
```

Comment:
This routine doesn't seem to be used anywhere

Reply:
All of the <package>_init_interface routines are invoked from the macro gluing for the H5_PACKAGE_INIT macro in H5private.h